.travis.yml

```diff
@@ -15,7 +15,7 @@ before_script:
 # install formatter
 - rustup component add rustfmt-preview
 # install linter
-- rustup component add clippy-preview
+#- rustup component add clippy-preview
```

Comment:
Were these removals intentional?

Reply:
Sadly, yes. Nightly currently does not support Clippy.